beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -231,30 +225,39 @@ func (vs *Server) BuildBlockParallel(ctx context.Context, sBlk interfaces.Signed
 		vs.setBlsToExecData(sBlk, head)
 	}()
 
-	localPayload, overrideBuilder, err := vs.getLocalPayload(ctx, sBlk.Block(), head)
-	if err != nil {
-		return status.Errorf(codes.Internal, "Could not get local payload: %v", err)
-	}
+	winningBid := primitives.ZeroWei
+	var bundle *enginev1.BlobsBundle
+	if slots.ToEpoch(sBlk.Block().Slot()) >= params.BeaconConfig().BellatrixForkEpoch {
```

Comment:
dumb question maybe, do we have to check in this way to account for forks? instead of checking the block/state version?